src/chainparams.cpp

```diff
@@ -280,11 +278,11 @@ class CRegTestParams : public CTestNetParams {
         nEquihashK = K;
         genesis.nTime = 1296688602;
```

Comment:
This timestamp is 2011-02-02 23:16:42. Does not block (we're resetting again, I think).